addons/web/static/tests/mobile/burger_menu_tests.js

```diff
@@ -122,3 +122,37 @@ QUnit.test("Burger menu closes when an action is requested", async (assert) => {
     assert.containsNone(document.body, ".o_burger_menu");
     assert.containsOnce(document.body, ".o_kanban_view");
 });
+
+QUnit.test("Burger menu closes when click on menu item", async (assert) => {
+    assert.expect(4);
+    serverData['actions']['1']['target'] = 'new';
```

Comment:
serverData.actions['1'].target would be more js like

Reply:
or even `serverData.actions[1]`

Reply:
@SimonGenin @aab-odoo I have done all changes.Thanks for your suggestion.

cc @IT-Ideas